### PR TITLE
add notes for known failures

### PR DESCRIPTION
**Pull request by mambocab**
@knifewine can you have a quick look? this is just more information in the `notes` fields of some `known_failure`s.

### CODE REVIEW

**Comment by mambocab** [comment]
ping @knifewine and/or @ptnapoleon Anything in here wrong?

**Comment by ptnapoleon** [comment]
+1

File: index_summary_upgrade_test.py [hunk 1]
```diff
@@ -8,7 +8,8 @@
 class TestUpgradeIndexSummary(Tester):
 
     @known_failure(failure_source='cassandra',
-                   jira_url='https://issues.apache.org/jira/browse/CASSANDRA-11127')
+                   jira_url='https://issues.apache.org/jira/browse/CASSANDRA-11127',
+                   notes='Fails on and on Linux on 3.0 branch and on Windows')
```
Comment (knifewine) [inline index_summary_upgrade_test.py]: note's a little confusing.

**Comment by knifewine** [comment]
+1 after nit

File: index_summary_upgrade_test.py (hunk 1 above)
Comment (mambocab) [inline index_summary_upgrade_test.py]: fails, on and on and on

Comment (knifewine) [inline index_summary_upgrade_test.py]: Is this just saying it fails every time on linux?

Comment (ptnapoleon) [inline index_summary_upgrade_test.py]: This note is totally incomprehensible, good point Russ

Comment (mambocab) [inline index_summary_upgrade_test.py]: Naw, extra 'and on'. Pushing a fix in a couple.

**Comment by mambocab** [comment]
Cheers, merging once Travis is happy.

**Pull request merged**